store/sqlstore/channel_store_categories.go

```diff
@@ -606,7 +606,8 @@ func (s SqlChannelStore) UpdateSidebarCategoryOrder(userId, teamId string, categ
 	return nil
 }
 
-func (s SqlChannelStore) UpdateSidebarCategories(userId, teamId string, categories []*model.SidebarCategoryWithChannels) ([]*model.SidebarCategoryWithChannels, []*model.SidebarCategoryWithChannels, error) {
+// `teamId` is replaced with _ to pass unused param lint
+func (s SqlChannelStore) UpdateSidebarCategories(userId, _ string, categories []*model.SidebarCategoryWithChannels) ([]*model.SidebarCategoryWithChannels, []*model.SidebarCategoryWithChannels, error) {
```

Comment:
Maybe we can change the store interface for this one and the next one. @reflog any thoughts on this?